spec/spec_helper.rb

```diff
@@ -8,6 +8,7 @@
 require 'pry'
 
 Dir[File.join(File.dirname(__FILE__), 'support', '**', '*.rb')].each { |f| require f }
+Dir[File.join(File.dirname(__FILE__), '**', '*_examples.rb')].each { |f| require f }
```

Comment:
Can we drop this in favour of placing the shared examples in `spec/support/shared_examples/` instead?